packages/varnish/src/components/logos/AI2Projects.tsx

```diff
@@ -204,3 +174,252 @@ export function Incubator({ className, width, height }: IconProps) {
+            fill="#b5f0ff"
+          />
+          <path
+            data-name="Path 213"
+            d="M22.6 42.67l1.55-12.93c0-.06-.06-.12-.14-.12h-6.26c-.11 0-.18-.09-.12-.16l14.3-18.15c.08-.1.3 0 .26.08l-4.43 15c0 .06 0 .13.14.13H35c.1 0 .17.08.12.15l-12.28 16c-.05.16-.25.11-.24 0z"
+            fill="#265ed4"
+          />
+        </g>
+      </g>
+    </svg>
+  );
+}
+
+export function Wildlands({ width, height }: IconProps) {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 150 181"
+      width={width || defaults.SIZE}
+      height={height || defaults.SIZE}
+    >
+      <defs>
+        <clipPath id="clip0_76_62">
+          <rect width="150" height="180" fill="white" transform="translate(0 0.5)" />
+        </clipPath>
+      </defs>
+      <g clipPath="url(#clip0_76_62)">
+        <path
+          d="M6 110.05C6 145.59 37 174.5 75 174.5C113 174.5 144 145.59 144 110.05V6.5H6V110.05Z"
+          fill="#C0EEFD"
+        />
+        <path
+          d="M0 0.5V110.05C0 147.9 31.78 178.77 71.63 180.42C72.75 180.47 73.87 180.5 75 180.5C76.13 180.5 77.25 180.5 78.37 180.42C94.2134 179.818 109.492 174.378 122.15 164.83C139.15 151.91 149.98 132.19 149.98 110.07V0.5H0ZM144 110.05C144 128.56 135.59 145.26 122.17 157.05V115.5L132.5 125.83L138.29 120.04L122.17 103.92V99.12L132.5 109.5L138.29 103.71L122.17 87.57V82.78L132.5 93.11L138.29 87.32L118.07 67.1L97.85 87.32L103.64 93.11L114 82.78V87.57L97.85 103.7L103.64 109.49L114 99.16V103.96L97.85 120.08L103.64 125.87L114 115.5V163.15C103.535 169.852 91.5095 173.722 79.1 174.38V97.87L89.43 108.2L95.22 102.41L79.1 86.28V81.5L89.43 91.82L95.22 86.03L95.12 85.92L95.22 85.82L79.1 69.7V65.11L89.43 75.44L95.22 69.65L95.12 69.54L95.22 69.44L79.1 53.32V48.73L89.43 59.06L95.22 53.27L95.12 53.16L95.22 53.06L79.1 36.93V32.14L89.43 42.5L95.22 36.71L75 16.5L54.78 36.68L60.57 42.5L70.9 32.14V36.93L54.78 53.06L54.88 53.16L54.78 53.27L60.57 59.06L70.9 48.73V53.32L54.78 69.44L54.88 69.54L54.78 69.65L60.57 75.44L70.9 65.11V69.7L54.78 85.82L54.88 85.92L54.78 86.03L60.57 91.82L70.9 81.5V86.29L54.78 102.41L60.57 108.2L70.9 97.87V174.38C58.4941 173.734 46.4685 169.878 36 163.19V113.93L46.34 124.26L52.15 118.5L36 102.34V97.5L46.34 107.84L52.13 102.05L36 85.96V81.16L46.36 91.5L52.15 85.71L36 69.58V64.78L46.36 75.12L52.15 69.33L31.93 49.11L11.71 69.33L17.5 75.12L27.83 64.78V69.58L11.71 85.71L17.5 91.5L27.83 81.16V85.96L11.71 102.09L17.5 107.88L27.83 97.5V102.3L11.71 118.5L17.5 124.29L27.83 113.96V157.06C14.41 145.26 6 128.56 6 110.05V6.5H144V110.05Z"
+          fill="#3360CD"
+        />
+      </g>
+    </svg>
+  );
+}
+
+export function Reviz({ width, height, mono }: IconProps) {
+  if (mono) {
```

Comment:
lets swap to poly so that our existing mono assets continue to work as defined in external apps

Reply:
https://github.com/allenai/varnish-mui/pull/125